.github/workflows/build_and_test.yml

```diff
@@ -23,8 +23,8 @@ on:
   push:
     branches:
     - '**'
-    - '!branch-*.*'
```

Comment:
Think aloud here, if we enable the all branch validation in each branch push event, that means `branch-*.*` will use the regular (hadoop3/java 8). 

1). should we add some configure for each maintaines branches in `configure-jobs`?

2). BTW, after 1), I guess some cron jobs can be replaced by this branch push event job. 
```
    # branch-3.2
    - cron: '0 7 * * *'
```

Reply:
> Think aloud here, if we enable the all branch validation in each branch push event, that means `branch-*.*` will use the regular (hadoop3/java 8) to run ci in each commits in `branch-*.*` 

I think other branches won't be affected - . branch-3.2 and branch-3.3 has `**` pattern. I referred to https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#about-workflow-triggers

Reply:
Yep, I was confused by `**`, it means `current branch` for each branch.